README.md

```diff
@@ -92,3 +92,5 @@ cd sos-client
 ## Contributors
 
 - [Elochukwu Okafor (Team Lead)](https://github.com/dev-xero)
+- [Nwolisa Vincent (Assistant Team Lead)]
```

Comment:
This is very nice, good work bro. You can add your github link inside those parentheses after the square brackets like I did in mine.